documentation/staging/content/userguide/managing-domains/accessing-the-domain/port-forward.md

```diff
@@ -63,19 +63,52 @@ This behavior depends on your version and domain resource configuration:
   [schema](https://github.com/oracle/weblogic-kubernetes-operator/blob/main/documentation/domains/Domain.md).
 
 * For Istio-enabled domains running Istio versions prior to 1.10,
-  the operator already adds a
-  network channel with a `localhost` listen address for each
-  existing port. This means that no additional configuration is required
-  to enable port forwarding when Istio is enabled.
+  you must add an additional network channel to the WebLogic Administration Server 
+  configured with the following attributes:
+  * Protocol defined as `t3`.
+  * Listen address defined with `localhost`.
```

Comment:
Have you tested this? I'm pretty sure the introspector WLST generates overrides every listen address on every channel to force it to "DOMAINUID-WLSERVERNAME".

Reply:
I did test it.  This is  only for when you are Istio enabled AND Istio versions prior to 1.10.  In the introspector code, we customize naps to set the listen address to 'localhost/127.0.0.1' when Istio is enabled AND Istio requires localhost bindings:

def customizeNetworkAccessPoint(self, nap, listen_address):
    # Don't bother 'add' a nap listen-address, only do a 'replace'.
    # If we try 'add' this appears to mess up an attempt to 
    #   'add' PublicAddress/Port via custom sit-cfg.
    # FWIW there's theoretically no need to 'add' or 'replace' when empty
    #   since the runtime default is the server listen-address.

    istio_enabled = self.env.getEnvOrDef("ISTIO_ENABLED", "false")
    istio_use_localhost_bindings = self.env.getEnvOrDef("ISTIO_USE_LOCALHOST_BINDINGS", "true")
    if istio_enabled  == 'true' and istio_use_localhost_bindings == 'true':
      listen_address = '127.0.0.1'

Reply:
So technically, it doesn't matter what you set the listen address to?  Why do the new instructions say to set "localhost"?  We don't normally advise setting the LA to anything IIRC.

Reply:
Yes, technically doesn't matter.  I just added the instructions as it does match well in understanding why the channel has to bind listen address as 'localhost' when using Istio versions prior to v1.10 (i.e. istio proxying data to 'localhost').  I'm fine with removing the advice of setting the listen address to 'localhost', if that's the preference.

Reply:
OK - now I get it - thanks. I guess it's fine, but I prefer something like:

- Listen address defined with `localhost`. (Note that setting the address to `localhost` is solely for self documenting purposes. The address can be set to any value, and the operator will override it to the required value regardless.)

Pre-approving - leaving it up to you to add the additional wording.